checks/system/win32.py

```diff
@@ -178,6 +183,13 @@ def check(self, agentConfig):
             pct_usable = float(usable) / total
             self.save_sample('system.mem.pct_usable', pct_usable)
 
+        page = psutil.virtual_memory()
+        if page.total is not None:
```

Comment:
The answer is, in short, we can't because they're different. I cannot, at this moment, articulate the exact difference. I'm pretty sure that `Win32_PerfRawData_PerfOS_Memory` does not give you the actual pagefile usage, which is what we're trying to get here.